contracts/scripts/lifebank.sh

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env bash
+EOS_PRIV_KEY=# TODO: write priv key here
+EOS_PUB_KEY=# TODO: write pub key here
+
+set -e
+
+unlock_lifebank_wallet() {
+    echo 'Unlocking Wallet...'
+    cleos wallet unlock -n lifebank --password $(cat ./secrets/lifebank_wallet_password.txt) ||
+        echo "Wallet has already been unlocked..."
+}
+
+create_lifebank_wallet() {
+    echo 'Creating Wallet...'
+    cleos wallet create -n lifebank --to-console |
+        awk 'FNR > 3 { print $1 }' |
+        tr -d '"' \
+            >./secrets/lifebank_wallet_password.txt
+    cleos wallet open -n lifebank
+    unlock_lifebank_wallet
+    cleos wallet import -n lifebank --private-key $EOS_PRIV_KEY
+}
+
+create_lifebank_accounts() {
+    echo 'Creating Lifebank Accounts...'
+    lifebank_accounts=(
+        "consent2life"
+        "lifebankcode"
+        "lifebankcoin"
+        "lbacccreator"
+        "sponsprueba1"
+        "donorprueba1"
+    )
+
+    for account in "${lifebank_accounts[@]}"; do
+        echo "Creating lifebank account '$account'"
+
+        keys=($(cleos create key --to-console))
+        pub=${keys[5]}
+        priv=${keys[2]}
+
+        cleos wallet import -n lifebank --private-key $priv
+        mkdir -p ./secrets
+        echo $pub >./secrets/$account.pub
+        echo $priv >./secrets/$account.priv
+        cleos -u https://lifebank.eosio.cr system newaccount eosio \
+            --transfer $account \
+            $EOS_PUB_KEY \
+            --stake-net "10000.0000 EOS" \
+            --stake-cpu "10000.0000 EOS" \
+            --buy-ram-kbytes 8192
+    done
+}
+
+assign_resources() {
+    cleos -u https://lifebank.eosio.cr get account lbacccreator 
+    cleos -u https://lifebank.eosio.cr system buyram eosio lbacccreator "10000.0000 EOS"
+    cleos -u https://lifebank.eosio.cr get account lbacccreator 
+    cleos -u https://lifebank.eosio.cr transfer eosio lbacccreator "10000000.0000 EOS" "Lifebank initial funding"
```

Comment:
Not sure how useful it would be but maybe we can set these  values as parameters?

Reply:
Maybe the endpoint will be a good one